services/background-worker/src/Wire/BackgroundWorker/Env.hs

```diff
@@ -47,9 +47,9 @@ data Env = Env
     galley :: Endpoint,
     brig :: Endpoint,
     defederationTimeout :: ResponseTimeout,
-    remoteDomains :: IORef FederationDomainConfigs,
-    remoteDomainsChan :: Chan FederationDomainConfigs,
-    backendNotificationMetrics :: BackendNotificationMetrics,
+    remoteDomains :: IORef FederationDomainConfigs, -- TODO: remove
+    remoteDomainsChan :: Chan FederationDomainConfigs, -- TODO: remove
+    backendNotificationMetrics :: BackendNotificationMetrics, -- TODO: move into BackendNotificationsConfig?
```

Comment:
This is not a config so it cannot be moved in there. Merging them in some way would create hierarchies which just make things more painful to access, so I would leave it like this.

Reply:
(i noticed that, too.  :)